cernopendata/templates/cernopendata_records_ui/records/record_detail.html

```diff
@@ -195,6 +195,14 @@ <h2>Methodology</h2>
                 {% if step.get("global_tag") %}
                     Global Tag: {{ step.global_tag }}<br>
                 {% endif %}
+                {% if step.get("output_dataset") %}
+                    Output dataset: {{ step.output_dataset }}<br>
+                {% endif %}
+                {% if step.get("note") %}
+                    Note: {{ step.note }}<br>
```

Comment:
- [ ] `note` contains HTML, so we need `safe`

- [ ] Configuration files should be displayed first, then output datasets, then note

Reply:
One more thing: `To get the exact LHE and generator's parameters, see <a href=\"/docs/cms-mc-production-overview\"> CMS Monte Carlo production overview</a>. #finding-the-generator-parameters` the final anchor `#finding...` should be inside the `<a href="...">` link